src/WebJobs.Script/Environment/EnvironmentSettingNames.cs

```diff
@@ -34,6 +34,7 @@ public static class EnvironmentSettingNames
         public const string AppInsightsAgent = "APPLICATIONINSIGHTS_ENABLE_AGENT";
         public const string FunctionsExtensionVersion = "FUNCTIONS_EXTENSION_VERSION";
         public const string FunctionWorkerRuntime = "FUNCTIONS_WORKER_RUNTIME";
+        public const string MetadataProviderTimeoutInSeconds = "METADATA_PROVIDER_TIMEOUT_IN_SECONDS";
```

Comment:
This is only used to set the timeout value from tests. For normal use case, it will use the default value (30 seconds).

Reply:
I switched to a property in the latest iteration pushed.